### PR TITLE
Adding dbus-tools + other fixes

### PR DESCRIPTION
**Pull request by plauche**
* Adds dbus daemon plus init/config files
* Corrects BBB and iOBC toolchain paths
* Updates `provision-test` to properly test for BBB and iOBC toolchains
* Fixes a broken `apt-get install`
* Removes deprecated python repository
* Installs python dependency (cryptography) at non broken version

### CODE REVIEW

File: base/bin/dbus_setup.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```
Comment (kyleparrott) [inline base/bin/dbus_setup.sh]: Is this file supposed to be run automatically or just let a dev run it manually as they please?

Comment (kyleparrott) [reply base/bin/dbus_setup.sh]: Also in testing locally does this file persist across provisioning the `kubos-dev` box? i.e. Does the home directory from provisioning the `kubos-dev` box overwrite the home directory from the base box?

Comment (plauche) [reply base/bin/dbus_setup.sh]: This file is intended to be run by a dev manually. I didn't think we should have it start automatically, at least not yet. Re the files being preserved by the kubos-dev box, they should be preserved, we were already copying over four other files in the same way.

Comment (kyleparrott) [reply base/bin/dbus_setup.sh]: I don't think we copy those files over the same way. They're mounted into the home directory, then moved out in later provisioning steps:

mounted here: 
`config.vm.provision "file", source: "./bin/minirc.msp430", destination: "~/minirc.msp430"`
https://github.com/kubostech/kubos-vagrant/blob/master/base/Vagrantfile#L11

and moved here: 
`mv /home/vagrant/minirc.msp430 /etc/minicom/minirc.msp430`
https://github.com/kubostech/kubos-vagrant/blob/master/base/script/privileged-provision.sh#L69

File: base/bin/dbus_setup.sh [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+# Start D-Bus system daemon
+dbus-daemon --config-file=~/kubos-dbus.conf --fork --nopidfile --address=unix:path=/tmp/kubos
+
+export DBUS_SESSION_BUS_ADDRESS "unix:path=/tmp/kubos"
+export DBUS_STARTER_BUS_TYPE "session"
```
Comment (CatGarab) [inline base/bin/dbus_setup.sh]: Need equals signs

File: base/bin/dbus_setup.sh [hunk 3]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# Start D-Bus system daemon
+dbus-daemon --config-file=~/kubos-dbus.conf --fork --nopidfile --address=unix:path=/tmp/kubos
```
Comment (CatGarab) [inline base/bin/dbus_setup.sh]: For some reason `~/kubos-dbus.conf` resulted in that file not being found, but removing `~/` fixed it.

**Comment by plauche** [comment]
@kyleparrott this is ready for review and should be good now. However...it looks like there is an issue with one of the newer versions of git (2.15+) and GitPython which causes some weird errors when provisioning the kubos-dev box. We could either wait for them to be resolved or try rolling back versions of packages. Thoughts?

For reference:
  - https://github.com/msiemens/PyGitUp/issues/65
  - https://github.com/gitpython-developers/GitPython/pull/689

**Comment by kyleparrott** [comment]
It's always something.. I'm skeptical that gitPython issue will be resolved by Monday. 

Can we install an older version of `git` in the mean time?

**Pull request merged**